pkg/util/validation/errors_test.go

```diff
@@ -28,7 +28,12 @@ func TestNewMetadataUnitTooLongError(t *testing.T) {
 
 func TestNewMaxQueryLengthError(t *testing.T) {
 	err := NewMaxQueryLengthError(time.Hour, time.Minute)
-	assert.Equal(t, "the query time range exceeds the limit (query length: 1h0m0s, limit: 1m0s) (err-mimir-max-query-length). To adjust the related per-tenant limit, configure -store.max-query-length, or contact your service administrator.", err.Error())
+	assert.Equal(t, "the (sharded) query time range exceeds the limit (query length: 1h0m0s, limit: 1m0s) (err-mimir-max-query-length). To adjust the related per-tenant limit, configure -store.max-query-length, or contact your service administrator.", err.Error())
+}
+
+func TestNewTotalMaxQueryLengthError(t *testing.T) {
+	err := NewMaxTotalQueryLengthError(time.Hour, time.Minute)
+	assert.Equal(t, "the total unsharded query time range exceeds the limit (query length: 1h0m0s, limit: 1m0s) (err-mimir-max-total-query-length). To adjust the related per-tenant limit, configure -store.max-total-query-length, or contact your service administrator.", err.Error())
```

Comment:
For the same reason, not correct to say "unsharded" here. I would just remove that wording.